services/cd-service/pkg/repository/transformer.go

```diff
@@ -2190,6 +2190,72 @@ func (c *ExtendAAEnvironment) Transform(
 	return fmt.Sprintf("Successfully added ArgoCD configuration '%s'", c.Environment), nil
 }
 
+type DeleteAAEnvironmentConfig struct {
+	Authentication          `json:"-"`
+	Environment             types.EnvName    `json:"env"`
+	ConcreteEnvironmentName types.EnvName    `json:"concreteEnvName"`
+	TransformerEslVersion   db.TransformerID `json:"-"` // Tags the transformer with EventSourcingLight eslVersion
+}
+
```

Comment:
please add a line that ensures this struct implements transformer, something like this:

```suggestion
var _ Transformer = (*DeleteAAEnvironmentConfig)(nil) // ensure we implement the interface
```